frontend/src/components/NoAuth.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import useAuth from "../hook/useAuth";
+import {Navigate} from "react-router-dom";
+
+export default function NoAuth({
+    children,
+    redirect
+                               }:{
```

Comment:
What are these space characters used for?

Reply:
Helps me get the syntax right, but I forgot to take them out here!
_Changed_